### ISSUE TITLE
Are there any plans to release pretrained model weights, as you did for improved-diffusion?

### ISSUE THREAD

**Issue by crowsonkb** [openai/guided-diffusion#1]
There are many people interested in plugging gradients from CLIP into a diffusion model.

**Comment by aicrumb** [comment]
you can use the models from improved-diffusion with this, you just have to train your own classifier (in my limited experience, i would recommend not to use the imagenet or cifar models, just the lsun bedroom ones 👍 i've been having some complication getting them to work at all) you can plug clip into it right now if you know what you're doing, I've been trying to figure out how for days.

**Comment by afiaka87** [comment]
We would love to experiment with this in the pipeline over at the the [DALLE-pytorch repo](https://github.com/lucidrains/DALLE-pytorch/) as well. We've made tons of progress replicating the OpenAI paper and we're already having a ton of success using the released weights fro the Taming-Transformers research - which I believe @crowsonkb and group have been highly successful in "guiding" various vision transformers via the few released pretrained CLIP models. This work is fascinating and seems to not require all that much compute.

Would be an interesting experiment to see if results improve!

**Comment by betterze** [comment]
any reply about the pretrained models? Thx

**Comment by unixpickle** [comment]
Model checkpoints have been released and are linked in the README. Please be sure to check out the [model card](https://github.com/openai/guided-diffusion/blob/main/model-card.md) for intended use and limitations.

**Comment by crowsonkb** [comment]
Thank you so much for releasing the weights! I released a CLIP guided diffusion Colab notebook this morning which uses the 256x256 unconditional model. https://colab.research.google.com/drive/12a_Wrfi2_gwwAuN3VvMTwVMz9TfqctNj
